pavelib/assets.py

```diff
@@ -597,6 +607,19 @@ def process_npm_assets():
     """
     Process vendor libraries installed via NPM.
     """
+    def copy_vendor_library(library, skip_if_missing=False):
```

Comment:
This approach is going to break when we upgrade to NPM 3+ (or, at least, will be a reason we can't support <3 and >3 simultaneously).  Is there a better way we can do (or not do) this whole copying to `/vendor` thing? It seems like at a minimum requirejs (and the pre-r.js django asset stuff, too, potentially) should really be able to look for modules in `node_modules` on its own.